src/chart_types/xy_chart/utils/axis_utils.ts

```diff
@@ -393,14 +393,16 @@ export function getLeftAxisMinMaxRange(chartRotation: Rotation, height: number)
       return { minRange: 0, maxRange: height };
   }
 }
-
+export function getIntegerTicks(scale: Scale, hasInteger: boolean = false) {
```

Comment:
Sorry to nitpick here but I think this function should be `getTicks` because it returns both ticks and integer ticks. Then you can pass the `integersOnly` boolean as an argument.

```suggestion
export function getTicks(scale: Scale, integersOnly: boolean = false) {
```

Reply:
I like this! I'm still working on better naming of variables etc so this is very appreciated. Thanks!

Reply:
I takes practice and there are sometimes not great options.

Reply:
commit 607af9b for changes